wgsl/README.md

```diff
@@ -0,0 +1,19 @@
+# WebGPU Shading Language Specification
+
+## Generating the specification
+
+The specification is written using [Bikeshed](https://tabatkins.github.io/bikeshed).
+
+If you have bikeshed installed locally, you can generate the specification with:
+
+```
+make
+```
+
+This simply runs bikeshed on the `index.bs` file.
```

Comment:
Nitpick:   Avoid "simply".  It's not simple the first time you do it.  Installing bikeshed might be part of that first time.

Reply:
Don't need to install Bikeshed. https://api.csswg.org/bikeshed/

Reply:
Installing bikeshed is optional. You can use the `make online` variant if you don't want to install bikeshed.